src/CommunityVoices/App/Website/Presentation/Module/Form/Quote.xslt

```diff
@@ -40,37 +37,28 @@
-                                        <xsl:when test="domain/quote != ''">
-                                          update
-                                        </xsl:when>
-                                        <xsl:otherwise>
-                                          upload
-                                        </xsl:otherwise>
-                                      </xsl:choose>
-                                    </h1>
-                                    <ul style="margin-bottom: 0.5rem;">
-                                        <xsl:apply-templates select="domain/errors/*" />
-                                        <xsl:apply-templates select="domain/upload/errors/*" />
-                                    </ul>
-                                </div>
-                            </div>
+                            <xsl:variable name="updateOrUpload">
+                                <xsl:choose>
+                                  <xsl:when test="domain/quote != ''"> Update </xsl:when>
+                                  <xsl:otherwise> Upload </xsl:otherwise>
+                                </xsl:choose>
+                            </xsl:variable>
+                            <xsl:variable name="errorsList">
+                                <xsl:for-each select="domain/upload/errors/*|domain/errors/*">
+                                    <item> <xsl:value-of select="."/> </item>
+                                </xsl:for-each>
+                            </xsl:variable>
+                            <xsl:call-template name="card">
+                                <xsl:with-param name="title" select="concat('Some Errors Prevented ', $updateOrUpload)"/>
+                                <!-- Note, pass in message with <item> tags (see "$errorsList") -->
+                                <xsl:with-param name="message" select="$errorsList"/>
+                            </xsl:call-template>
                         </xsl:when>
                         <xsl:when test="domain/repeatedQuoteErrorFree != ''">
-                            <div class="card" style="margin-bottom: 16px;">
-                                <div class="card-body">
-                                    <h1 class="h4 mb-4 font-weight-normal" style="margin-bottom: 0.5rem !important;">
-                                      Success!
-                                    </h1>
-                                    <ul style="margin-bottom: 0.5rem;">
-                                        <li><xsl:value-of select="domain/repeatedQuoteErrorFree"/></li>
-                                    </ul>
-                                </div>
-                            </div>
+                            <xsl:call-template name="card">
+                                <xsl:with-param name="title"> Success! </xsl:with-param>
+                                <xsl:with-param name="message"><item>Your quote was succesfully uploaded.</item></xsl:with-param>
```

Comment:
As you set this in PHP, might as well use that value, no?

Reply:
Great point. I'll change this right now

Reply:
No need to call it stupid